Tests/docker_native_image_config.json

```
@@ -143,12 +143,21 @@
             "native:dev",
             "native:candidate"
          ]
+      },
+      {
+         "id": "Intezer v2",
+         "reason": "Lint and UT depend on intezer-sdk package",
+         "ignored_native_images": [
+            "native:dev",
+            "native:ga",
+            "native:maintenance"
```

```suggestion
            "native:ga",
```

try to fix it for `native:dev` if possible, if not add it back

---

https://github.com/demisto/content/pull/28714/commits/de7d150d247c98c15031a9164da88511913a6153